### ISSUE TITLE
Paired image training

### ISSUE THREAD

**Issue by tarungog** [deepinstruction/deepnude_official#25]
Couldn't you easily get a database full of paired clothed/nude same-pose images from something like https://www.reddit.com/r/OnOff/ ? If you extract the massive amount of data on this subreddit (probably 10^5 images on here), you could likely create a much more compelling and less fragile model than the one in this app. Anyone want to try?

**Comment by tarungog** [comment]
Can you also release the datasets @deepinstruction

**Comment by c0mr4de** [comment]
I'm not entirely sure if this is how DeepNude works, but from my understanding, that's not how thedel works. I think the model works by marking the clothes and filling it with nudes. You can only "let the AI see" nude pictures or the model will also try to fill it with another clothes.

If you want to train the model, I think you can just go to PornHub, look for stripping videos, and split it into separate frames. You'll get more training resources

**Comment by tarungog** [comment]
yes DeepNude works differently. It's a clever design but also, I think, overly complicated. If we could simply pix2pix straight from clothed to nude, we'd be doing much better. PH stripping frames are unlikely to have the star maintain the same pose in both nude and clothed, but this is very common on r/onoff
